src/scripts.js

```diff
@@ -28,13 +28,7 @@ let hotel;
 // Event Listeners
 
 window.addEventListener('load', () => {
-  getData()
-    .then((data) => {
-      hotel = new Hotel (data[0].customers);
-      hotel.generateRooms(data[1].rooms);
-      hotel.generateBookings(data[2].bookings);
-      renderTypes()
-    });
+  resolveData();
```

Comment:
I like how you fetch all of the data while the user is logging in! Great job keeping the waiting times down!